compiler-rt/lib/interception/interception_win.cpp

```diff
@@ -643,6 +664,8 @@ static size_t GetInstructionSize(uptr address, size_t* rel_offset = nullptr) {
     case 0x7980:  // 80 79 YY XX  cmp BYTE ptr [rcx+YY], XX
       return 4;
 
+    case 0xB841:  // 41 B8 XX XX XX XX : mov r8d, XX XX XX XX
```

Comment:
This looks wrong. The second operand is an immediate, not a relative offset, so it shouldn't be leading to a branch that sets `rel_offset`.